pkgs/tools/admin/okta-aws-cli/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "okta-aws-cli";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner  = "okta";
+    repo   = "okta-aws-cli";
+    rev    = "v${version}";
+    sha256 = "1qlc04fydcqnifbhrzdw63wq3ndzsyzp4s1bdvfhvizbhdi03mpp";
+  };
+
+  vendorSha256 = "1gzaadpf9zrci7yv88f434mvc7gralb7dj6wl6r4vsv3qk291680";
+
+  doCheck = false;
```

Comment:
```suggestion
```

it doesn't seem to fail on my end, why the tests are disabled?

Reply:
removed it